moped-database/migrations/1606833580979_alter_table_public_moped_project_alter_column_project_priority/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."moped_project" ALTER COLUMN "project_priority" TYPE text;
```

Comment:
Is there a reason we are using text and not integers? integers are easier to index and more performant than text, but it is not a big problem whatsoever.

Reply:
JD wanted to switch to text for the user and in our code we have both integer and text so we could do the mutation with int. It was originally working fine in the old endpoint as int but with the new endpoint, I got a graphql error that it was expecting text so I changed it just to get everything working as a quick fix and so I could see if the other parts of the mutations were working.

Reply:
I think it will be fine... as long as we are not expecting a lot of different priorities :)